nixos/modules/services/web-apps/changedetection-io.nix

```diff
@@ -138,7 +138,8 @@ in
           StateDirectory = mkIf defaultStateDir "changedetection-io";
           StateDirectoryMode = mkIf defaultStateDir "0750";
           WorkingDirectory = cfg.datastorePath;
-          Environment = lib.optional (cfg.baseURL != null) "BASE_URL=${cfg.baseURL}"
+          Environment = [ "HIDE_REFERER=true" ]
```

Comment:
The `HIDE_REFERER` value set here breaks this app when I run it behind a proxy. I have to override it using the `environmentFile` option.  Otherwise changedetection-io returns an error `The referrer header is missing`:

```
<!doctype html>
<html lang=en>
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The referrer header is missing.</p>
```

Reply:
Can confirm, however I figured out that just adding `HIDE_REFERER=false` to `systemd.services.changedetection-io.serviceConfig.Environment` also fixes it if you don't want to use the environment file